src/fondant/pipeline/runner.py

```diff
@@ -126,3 +126,32 @@ def get_name_from_spec(self, input_spec: str):
         with open(input_spec) as f:
             spec = yaml.safe_load(f)
             return spec["pipelineInfo"]["name"]
+
+
+class SagemakerRunner(Runner):
+    def __init__(self):
+        self.__resolve_imports()
+
+    def __resolve_imports(self):
+        import boto3
```

Comment:
does this need to be added to `pyproject.toml`? Can we do a try/except with a valid message on which extras to install?

Reply:
Yes ! Will add it 

I guess its installed along with the sagemaker package but I will add it explicitly